notebooks/Model_Training_with_BERT.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>I recommend moving the code that initializes the variable <code>bert</code>  to this cell, so the reader doesn't have to scroll up to find it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>